Makefile

```diff
@@ -39,4 +39,12 @@ clean:
 	rm -rf .venv roles/external site.retry collections 
 	
 clean-all: clean
-	rm -rf .vagrant
\ No newline at end of file
+	rm -rf .vagrant
+
+check-righttoknow:
```

Comment:
This seems unrelated to the change described in the title. Since this repo is fairly new to being managed this way, I'm not strongly objecting to letting this through anyway, but I wanted to note that my general preference would be for unrelated changes to be in their own PRs

(It should also be noted that most of my experiences has involved tools like Google's Mondrian, or the open source clone Gerrit (https://www.gerritcodereview.com/), which both have tools that make it much easier to handle chains of pull requests than stock Git or Github PRs - so it's possible my preferences are not well aligned with the tooling in this case)

Reply:
i think there are "Release Chain" like tools in github for this. But agreed, separate things in separate PRs is better